src/Home/Home.tsx

```diff
@@ -38,7 +40,8 @@ const Home: React.FC = () => {
   </a>,
   <a
    href="https://www.active.com/outdoors/articles/5-tips-for-successful-altitude-acclimation"
-   target="_blank"
+	 target="_blank"
+	 rel="noopener noreferrer"
```

Comment:
This is good to know, thanks for bringing it up in chat.

Reply:
thank you!!